### PR TITLE
#208 Organisms - FOCardWithImage

### PR DESCRIPTION
**Pull request by bryantobing12**
### Detailed description of changes

- Need to update component to match the design

### Recommendations on how to test the changes

- Make sure the component match the design

### Checklist for developer

- [x] The code builds clean without any errors or warnings
- [x] The code does not contain commented out code
- [x] The code does not log anything to console
- [x] I have added unit test(s) to cover new code and successfully executed ran it
- [x] I have thoroughly tested the new code and any adjacent features it may affect

### Link to related issue(s)

https://app.zenhub.com/workspaces/flick-614155581501e1000f356a5d/issues/hzn-one/tms/208

#### Before

N / A

#### After

image

image

## Update
image
image

### CODE REVIEW

File: src/Components/Organisms/FOCardWithImage/index.tsx [hunk 1]
```diff
@@ -1,6 +1,5 @@
 import React, { memo } from "react";
 import { Card, Box } from "@mui/material";
-import CardActions from "@mui/material/CardActions";
```
Comment (gilangdarmawan) [inline src/Components/Organisms/FOCardWithImage/index.tsx:3]: Ada concern kenapa Card Actions di remove?

File: stories/Components/Organisms/FOCardWithImage/FOCardWithImage.stories.tsx [hunk 2]
```diff
@@ -1,230 +1,233 @@
...
-import { ComponentStory } from "@storybook/react";
+import { Meta, Story } from "@storybook/react";
+
 import { FAButton, FARadio } from "@Atoms";
 import { IFOCardWithImage } from "@Organisms/FOCardWithImage/FOCardWithImage.interfaces";
 import { FOCardWithImage } from "@Organisms/FOCardWithImage";
 
-const story = {
+const svg = (
+  <svg
```
Comment (gilangdarmawan) [inline stories/Components/Organisms/FOCardWithImage/FOCardWithImage.stories.tsx:9]: Possible pake component selain SVG ga? misal `` atau [next/image](https://nextjs.org/docs/api-reference/next/image) 

untuk dummy image yg di-show di storybook, selain SVG, bisa pake alternatif jpg atau png format. bisa ambil src dari [sini](https://dummyimage.com/250x180/e5e5e5/e5e5e5.jpg)

**Review by gilangdarmawan — CHANGES_REQUESTED** [review]

File: src/Components/Organisms/FOCardWithImage/index.tsx (hunk 1 above)
Comment (bryantobing12) [reply src/Components/Organisms/FOCardWithImage/index.tsx:3]: buat nyesuain padding sama spacing sih. `CardActions` itu beda container sama `CardContent` jadi padding nya bakal beda.

File: stories/Components/Organisms/FOCardWithImage/FOCardWithImage.stories.tsx (hunk 2 above)
Comment (bryantobing12) [reply stories/Components/Organisms/FOCardWithImage/FOCardWithImage.stories.tsx]: updated @gilangdarmawan

File: src/Components/Organisms/FOCardWithImage/index.tsx [hunk 3]
```diff
@@ -28,18 +27,25 @@ export const FOCardWithImage = memo((props: IFOCardWithImage) => {
...
             label={optionLabel || "optionLabel"}
-            testID="ControlLabel-test"
+            testID="form-control-label"
             control={option}
+            sx={{ mb: 3 }}
           ></FAFormControlLabel>
         )}
-        {svg}
+        <Box display="flex" justifyContent="center" width="100%">
+          <Box display="flex" mb={3} sx={{ width: "250px", height: "180px" }}>
```
Comment (gilangdarmawan) [inline src/Components/Organisms/FOCardWithImage/index.tsx:45]: I think it just needs one `<Box>` component. remove box in line 45. Add `mx="auto"` and `maxWidth="250px"` props in line 44 to centralize the child component.

**Review by gilangdarmawan — CHANGES_REQUESTED** [review]

File: src/Components/Organisms/FOCardWithImage/index.tsx (hunk 3 above)
Comment (bryantobing12) [reply src/Components/Organisms/FOCardWithImage/index.tsx]: updated @gilangdarmawan

**Review by gilangdarmawan — APPROVED** [review]

**Pull request merged**